packages/ember-glimmer/lib/renderer.js

```diff
@@ -367,8 +369,10 @@ class Renderer {
 
     try {
       this._renderRoots();
-    } finally {
+    } catch (e) {
```

Comment:
:( doesn't this bring back the pain while debugging?

Could we do this in the finally with a local variable?